modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -195,18 +228,44 @@ public void onConfigurationCommitted(RaftGroupConfiguration config, long lastApp
 
     @Override
     public void onSnapshotSave(Path path, Consumer<Throwable> doneClo) {
-        // TODO: implement, see https://issues.apache.org/jira/browse/IGNITE-22416
-        throw new UnsupportedOperationException("Snapshotting is not implemented");
```

Comment:
Could you please also port the comment from `PartitionListener#onSnapshotSave()`?